### PR TITLE
Create guide for GitHub web interface

### PR DESCRIPTION
**Pull request by tbodt**
I thought there were some problems with the existing guide. The command line is definitely the preferred method if you are using a Raspberry Pi, but most people have a web browser and would be more comfortable using that. Also, the explanation of using Git with the command line is only useful if you already know how to use Git with the command line.

So I made my own guide.

I'm open to suggestions for improvement.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*~
+.DS_Store
```
Comment (JuanitoFatas) [inline .gitignore]: IMHO, I think these better go to your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

File: GitCommitsForNonGitUsers.md [hunk 2]
```diff
@@ -1,42 +0,0 @@
...
-* `git push -f origin <BRANCHNAME>`
-
-## TO MERGE
-* `git log` to view the list of commits on the current branch
-* `git checkout master`
-* `git pull origin master` to be up to date
-* `git merge <BRANCHNAME>`
-* `git push origin master`
-* Delete branch from web UI
-* `git branch -d <BRANCHNAME>` to delete branch locally
```
Comment (JuanitoFatas) [inline GitCommitsForNonGitUsers.md]: This guide could consolidate into `CONTRIBUTING.md`?

File: github-web-interface.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# The GitHub Web Interface
+
+This is a guide to help those unfamiliar with git contribute to the TIL repository.
```
Comment (JuanitoFatas) [inline github-web-interface.md]: Suggestion: This guide is to help Non-Git friends to contribute to the TIL repository.

File: github-web-interface.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+ 6. Make a pull request.
+
+    ![Pull Request 1](img/pull-request-1.png)
+    ![Pull Request 2](img/pull-request-2.png)
+
+    Describe your change in the comment box.
+
+ 7. Wait for it to be merged with the main repository.
+
+You just contributed to the TIL repository. Make sure to tell all your friends.
```
Comment (JuanitoFatas) [inline github-web-interface.md]: Could indent this one for consistency.

**Comment by zamith** [comment]
@tbodt I agree that if you're not familiar with git or github, we probably will want to use the web interface. However, "The command line is definitely the preferred method if you are using a Raspberry Pi" is simply not true. A lot of people prefer the command line for a lot of different reasons.

Thanks for the PR.

**Comment by GolfyMcG** [comment]
@zamith Completely agree - I'm hardly deep into the Git game and it's much easier for me to manage forking and commiting from the command line. I do think there is a place for this guide though, but I would keep both.

File: github-web-interface.md (hunk 4 above)
Comment (tbodt) [inline github-web-interface.md]: I'm not indenting it because it's not part of the list.

**Comment by JoelQ** [comment]
@tbodt Thanks for contributing this guide! At thoughtbot we do all our git work via the command-line. I don't think we should remove the command-line guide (although I could see replacing it with a reference to https://github.com/thoughtbot/guides/tree/master/protocol/git)

Github already has an official guide to their web workflow. Thoughts on just linking to it? https://help.github.com/articles/github-flow-in-the-browser/

**Comment by tbodt** [comment]
@JoelQ I think my guide is better than the GitHub web workflow guide because it doesn't require clicking on anything. You just scroll through.

**Comment by thorncp** [comment]
The benefit of linking to GitHub's guide is that it will stay up-to-date when they inevitably make UI/UX changes. I'm also not keen on adding a handful of images to this otherwise plain text repo.

File: github-web-interface.md (hunk 4 above)
Comment (muescha) [inline github-web-interface.md]: it can be point `8.`

**Comment by muescha** [comment]
i like this short guide

it shows me that it is very easy to add new TIL

btw: the GitHub's guide looks on the first sight very complicated if they need to split it into 10 help pages. 

but we should add a link to the GitHub's guide if someone need it more detailed (and maybe with updated images)

File: github-web-interface.md (hunk 4 above)
Comment (tbodt) [inline github-web-interface.md]: @muescha I prefer it be separate.

**Comment by Keithbsmiley** [comment]
This content definitely doesn't feel right for this. Like @thorncp we'll never really be able to keep this up to date. I also feel like this isn't the first place someone who was new to GitHub would look for how to submit a PR to this repo. This seems like it would be better as a link to an existing guide or a blog post.

**Comment by JoelQ** [comment]
Thanks everyone for contributing to the discussion. We will not be adding a custom guide to GitHub's web UI.

**Pull request closed without merging**